core/iam_authenticator_test.go

```diff
@@ -388,7 +388,7 @@ func TestIamGetTokenSuccess(t *testing.T) {
 	assert.Nil(t, authenticator.synchronizedRequestToken())
 
 	// Force expiration and verify that we got the second access token.
-	authenticator.getTokenData().Expiration = GetCurrentTime() - 3600
+	authenticator.getTokenData().Expiration = GetCurrentTime() - 1
```

Comment:
No need to subtract 3600 (1 hour) from the current time to cause the "expiration" to occur.